.vscode/settings.json

```diff
@@ -1,4 +1,8 @@
 {
   "typescript.tsdk": "node_modules/typescript/lib",
-  "workbench.colorCustomizations": {}
-}
\ No newline at end of file
+  "workbench.colorCustomizations": {},
+  "editor.formatOnPaste": true,
```

Comment:
I don't think these properties should be added for libs, espetially these extension specific properties.

Reply:
If we do it for apps, why not do it for libs as well? If the repo is using prettier (it already has a config file), I think it would be good to have prettier as a recommended plugin for the workspace and to enable formatting for VSCode?